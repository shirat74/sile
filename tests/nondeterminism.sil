\begin[papersize=a4,class=book]{document}
\font[size=11pt,family=Gentium Book Basic]
\noindent{}Now that we understand some of what SILE is about and what it seeks to do, 
letâ€™s dive into SILE itself.

\end{document}