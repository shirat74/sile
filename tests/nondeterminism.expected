This is SILE 0.9.3-unreleased
<tests/nondeterminism.sil>Open file	tests/nondeterminism.pdf
Set paper size 	595.275597	841.8897729
Begin page
Mx 	49.40787
My 	105.98441
Set font 	Gentium Book Basic;11;200;normal;normal;
T	49 82 90	(Now)
Mx 	73.71946
T	87 75 68 87	(that)
Mx 	96.55937
T	90 72	(we)
Mx 	113.22789
T	88 81 71 72 85 86 87 68 81 71	(understand)
Mx 	170.11517
T	86 82 80 72	(some)
Mx 	197.93945
T	82 73	(of)
Mx 	210.77838
T	90 75 68 87	(what)
Mx 	237.54456
T	54 44 47 40	(SILE)
Mx 	260.92158
T	76 86	(is)
Mx 	272.07399
T	68 69 82 88 87	(about)
Mx 	302.11116
T	68 81 71	(and)
Mx 	323.05508
T	90 75 68 87	(what)
Mx 	349.82126
T	76 87	(it)
Mx 	360.49027
T	86 72 72 78 86	(seeks)
Mx 	388.93223
T	87 82	(to)
Mx 	402.12565
T	71 82 15	(do,)
Mx 	419.97581
T	79 72 87 183 86	(letâ€™s)
Mx 	442.80498
T	71 76 89 72	(dive)
Mx 	466.10143
T	76 81 87 82	(into)
Mx 	488.55999
T	54 44 47 40	(SILE)
Mx 	49.40787
My 	119.98441
T	76 87 86 72 79 73 17	(itself.)
[1] Mx 	267.42538
My 	732.62998
Set font 	Gentium;10;200;normal;normal;
T	17	(1)
End page
Finish

